From david-b@pacbell.net  Tue Apr 21 22:35:24 2009
From: Mark A. Greer <mgreer@mvista.com>
Date: Tue, 21 Apr 2009 20:52:54 -0700
Subject: USB: musb: Prevent multiple includes of musb.h
To: Greg KH <greg@kroah.com>
Cc: "Mark A. Greer" <mgreer@mvista.com>
Message-ID: <200904212052.54393.david-b@pacbell.net>
Content-Disposition: inline


From: Mark A. Greer <mgreer@mvista.com>

Add #ifndef to musb header file to prevent multiple inclusions.

Signed-off-by: Mark A. Greer <mgreer@mvista.com>
Signed-off-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb/musb.h |    5 +++++
 1 file changed, 5 insertions(+)

--- a/include/linux/usb/musb.h
+++ b/include/linux/usb/musb.h
@@ -7,6 +7,9 @@
  * key configuration differences between boards.
  */
 
+#ifndef __LINUX_USB_MUSB_H
+#define __LINUX_USB_MUSB_H
+
 /* The USB role is defined by the connector used on the board, so long as
  * standards are being followed.  (Developer boards sometimes won't.)
  */
@@ -101,3 +104,5 @@ extern int __init tusb6010_setup_interfa
 extern int tusb6010_platform_retime(unsigned is_refclk);
 
 #endif	/* OMAP2 */
+
+#endif /* __LINUX_USB_MUSB_H */
